ts/Series/Bubble/BubbleSeries.ts

```diff
@@ -719,6 +719,11 @@ addEvent(BubbleSeries, 'updatedData', (e): void => {
     delete e.target.chart.bubbleZExtremes;
 });
 
+// After removing series, delete the chart-level Z extremes cache.
+addEvent(BubbleSeries, 'remove', (e): void => {
+    delete e.target.chart.bubbleZExtremes;
```

Comment:
Do we have to use `e`? Isn't `this` (series) enough?

Reply:
Maybe a simple `if` check? Probably `e` doesn't have enough TS definition and is defined `as any`. In this case, it will be safer to add `if` checks and actually use better series TS

Reply:
Ok, I see it now. We are inside the arrow function so `this` context is not a `series`. Ok, let's keep it as it is, otherwise we would have to change it to a normal function.